module/Application/view/application/video/index.phtml

```diff
@@ -0,0 +1,16 @@
+<section class="features row">
+    <h1>Apigility Introduction</h1>
```

Comment:
Should this be updated or it's fine as the video is about Apigility?

Reply:
Updated in a subsequent patch.